# Subscriber
car_state_topic_name: /estimation/slam/state
#
ref_path_topic_name: /planning/ref_path

# Publisher
ctrl_cmd_topic_name: /control/pure_pursuit/control_command

predict_path_topic_name: /visual/pre_path

# MPC params:
weight:

  cte: 10
  epsi: 40
  v: 1000
  steer: 100
  throttle: 100
  steer_rate: 200
  throttle_rate: 1
  desire_vel: 10

# Pure Pursuit params:
desire_vel: 10

car_length: 1.88

look_ahead: 20

# other params:
N: 10

dt: 0.04

initial_velocity: 2

mission: trackdrive          # [trackdrive, skidpad, acceleration]

controller: lqr # [mpc, pure_pursuit,lqr,mpc_lat,mpc_kin,mpcc]

simulation: true

node_rate: 100   # [Herz]
